.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: ruby
+rvm:
+  - ruby-head
+  - 2.2
+  - 2.1
+  - 2.0.0
+  - 1.9.3
+branches:
+  only:
+    - master
```

Comment:
@tjwallace: no normally, it will still build PR (a bug has been closed in 2013 on that topic on travis-ci). It avoids building all single commits on other branch before they are pushed through PR.
In case there is an issue, we'll relax the branch constraint, but as far as I know it should be OK.